packages/rs-drive-abci/src/logging.rs

```diff
@@ -67,11 +72,76 @@ pub struct LogConfig {
 impl Default for LogConfig {
     fn default() -> Self {
         Self {
-            destination: String::from("stderr"),
-            verbosity: 0,
-            color: None,
+            destination: LOG_DESTINATION_STDOUT.to_string(),
```

Comment:
We can do it but then we need to change LogDestination::Bytes name to "bytes", or change destination config option to "ByteBuffer". What do you prefer?

Reply:
It works for StdErr and StdOut because these variants don't have params but doesn't work for Bytes since it requires Writer  so you can't do LogDestination::name(&LogDestination::Bytes);